websocketOutportDriver/client/wsConnClient.go

```diff
@@ -0,0 +1,55 @@
+package client
```

Comment:
we need to figure out how to test this thin wrapper over a library implementation. At least the basic stuff and the happy flows. Can be done in another PR